src/librustc_typeck/check/intrinsic.rs

```diff
@@ -67,15 +67,17 @@ fn equate_intrinsic_type<'tcx>(
 }
 
 /// Returns `true` if the given intrinsic is unsafe to call or not.
-pub fn intrinsic_operation_unsafety(intrinsic: &str) -> hir::Unsafety {
+pub fn intrinsic_operation_unsafety(intrinsic: Symbol) -> hir::Unsafety {
+    use rustc_span::symbol::sym::*;
```

Comment:
A glob import of `sym` pollutes the local namespace with a lot of lower-case constants. For this reason, it's not done anywhere else in the compiler. I realize that it's kind of a pain here, but we should uphold this example. It's more relevant in the other big match statement, which also has other code.

Reply:
yea... I knew that to be the right thing... but... laziness XD